modules/nf-core/fastp/main.nf

```diff
@@ -10,6 +10,7 @@ process FASTP {
     input:
     tuple val(meta), path(reads)
     path  adapter_fasta
+    val   save_trimmed_pass
     val   save_trimmed_fail
```

Comment:
why pass before fail?

Reply:
not sure i understand the issue here. users would probably be more interested in the trimmed reads that pass filter, so i put it before for priority.